src/ClusterBootstrap/deploy.py

```diff
@@ -2470,6 +2484,39 @@ def kubernetes_get_node_name(node):
         kube_node_name = node
     return kube_node_name
 
+def cordon(config, args):
```

Comment:
Duplication of the ones in `ctl.py`?

Reply:
actually not. the old pipeline didn't take config/args as input variables, and used global variables, which is a very important factor that lead to refactoring. 
difference lies where run_kubectl is invoked, in old pipeline, config/args are not passed to run_kubectl.